scripts/systemtap/kem/kemd.c

```diff
@@ -92,13 +90,15 @@ static int kemd_devs_create(void)
 	int err = 0;
 	int i;
 
-	for (i = 0; i < kemd_nr_cpus; i++) {
+	for (i = 0; i < kemd_nr_cpus; i++)
+	{
```

Comment:
Put the open brace `{` on the previous line.

Reply:
I have not  formatted that way, it is as it was earlier. there are  no of places where it need to be formatted in that file similarly. So I am not going to change that way. if at all it goes against the coding style , I will do later

Reply:
I don't understand your reply.
The original code is like this:
```
for (i = 0; i < kemd_nr_cpus; i++) {
```

Please keep this style according to "doc/coding-style":
```
 21   * braces placement and spacing is as following:
 22 
 23           if (condition) {
 24                   branch0;
 25           } else {
 26                   branch1;
 27           }
 28 
 29           func(arg0, arg1, ...);
 30 
 31           while (condition) {
 32                   body;
 33           }
 34 
 35           switch (expression) {
 36           case VALUE0:
 37                   branch0;
 38           case VALUE1:
 39                   branch1;
 40           ...
 41           default:
 42                   defaultbranch;
 43           }

```

Reply:
Formatted accordingly